netbox/ipam/querysets.py

```diff
@@ -148,7 +148,7 @@ def get_for_virtualmachine(self, vm):
 
         # Find all relevant VLANGroups
         q = Q()
-        site = vm.site or vm.cluster.site
+        site = vm.site or vm.cluster._site
```

Comment:
This should be adapted for the other scope types as well, as VLAN groups can also be assigned to different scopes.

Reply:
? Not sure I follow this - this is for VLAN, not VLANGroup and any higher scope assignment will filter down to _site - so _site should pick up all the assignments?